backend/src/ee/services/dynamic-secret/providers/totp.ts

```diff
@@ -4,20 +4,12 @@ import { HashAlgorithms } from "otplib/core";
 import { BadRequestError } from "@app/lib/errors";
 import { alphaNumericNanoId } from "@app/lib/nanoid";
 
-import { DynamicSecretTotpSchema, TDynamicProviderFns } from "./models";
+import { DynamicSecretTotpSchema, TDynamicProviderFns, TotpConfigType } from "./models";
 
 export const TotpProvider = (): TDynamicProviderFns => {
   const validateProviderInputs = async (inputs: unknown) => {
     const providerInputs = await DynamicSecretTotpSchema.parseAsync(inputs);
 
-    const urlObj = new URL(providerInputs.url);
-    const secret = urlObj.searchParams.get("secret");
-    if (!secret) {
-      throw new BadRequestError({
-        message: "TOTP secret is missing from URL"
-      });
-    }
-
```

Comment:
Can't we keep this validation for URL type based on the config type and add validation for the manual type? Would be helpful to catch these for users

Reply:
I removed this because we're now doing the validation in the API layer during post/patch requests